packages/extensions-registry/src/modules/search/lib/construct-text.ts

```diff
@@ -0,0 +1,28 @@
+import type { SearchOptions } from '../types/search-options.js';
+
+export const constructText = (options: Pick<SearchOptions, 'text' | 'extensionType' | 'author' | 'maintainer'>) => {
+	let text = options?.text ?? '';
+
+	if (options?.extensionType) {
+		/*
+		 * In versions < 10.8, the default tag was called directus-custom-x instead of
+		 * directus-extension-x. To allow those older extensions to still be found, we'll search for
+		 * both for the time being.
+		 *
+		 * @TODO remove `directus-custom-x` tag search
+		 */
+		text += ` keywords:"directus-extension-${options.extensionType}","directus-custom-${options.extensionType}"`;
```

Comment:
Apparently, quotes need to be around the whole keywords set:
```suggestion
		text += ` keywords:"directus-extension-${options.extensionType},directus-custom-${options.extensionType}"`;
```

https://registry.npmjs.org/-/v1/search?text=keywords:%22directus-extension-bundle%22,%22directus-custom-bundle%22
vs
https://registry.npmjs.org/-/v1/search?text=keywords:%22directus-extension-bundle,directus-custom-bundle%22